vulcanai2/plotters/utils.py

```diff
@@ -64,21 +71,28 @@ def remove_hooks(self):
         for h in self.hooks:
             h.remove()
 
-    def generate_gradients(self, input_x, target_class_list):
+    def generate_gradients(self, input_x, targets):
```

Comment:
this should also take the same input type... and we should be consistent in our naming of variables and parameters in regards to data/x/training and targets/y data.